src/components/logical-expression-builder/pipes/display-value.pipe.ts

```diff
@@ -0,0 +1,66 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { LogicalExpressionBuilderService } from '../services/logical-expression-builder.service';
+import { DisplayValueFunction } from '../interfaces/DisplayValueFunction';
+import { DateFormatterPipe } from '../../../pipes/date-formatter/date-formatter.pipe';
+
+/** Used to display the value of a condition. Contains a standard transform function for the basic five Input
+ * components. It can be overridden with the displayValueFunction input property. */
+@Pipe({ name: 'displayValue' })
+export class DisplayValuePipe implements PipeTransform {
+    private readonly _transformFunction: DisplayValueFunction;
+
+    constructor(private _lebService: LogicalExpressionBuilderService, private _dateFormatterPipe: DateFormatterPipe) {
+        this._transformFunction = this._lebService.getDisplayValueFunction();
+    }
+
+    transform(value: any, fieldType?: string, fieldData?: any): string {
```

Comment:
I would rather return an empty string instead of undefined.

Reply:
Thats fine if it returns an empty string, it currently returns a `string` or `undefined`, so the return type should be accurate to reflect the behaviour. If it can possibly return `undefined` then the return type should be `string | undefined`, if it returns an empty string as a fallback then `string` is fine for the return type.